dmapiclient/data.py

```diff
@@ -53,6 +53,7 @@ def get_audit_event(self, audit_event_id):
         )
 
     find_audit_events_iter = make_iter_method('find_audit_events', 'auditEvents')
+    find_audit_events.__name__ = str("find_audit_events_iter")
```

Comment:
Wrong object being edited, if you keep this way of doing things.

Reply:
huh. think my regex went awry. well caught.